src/Unic.ErrorManager.Core/Resources/Media/MediaRequestHandler.cs

```diff
@@ -141,10 +141,12 @@ protected override bool DoProcessRequest(HttpContext context)
             {
                 if (Sitecore.Configuration.Settings.RequestErrors.UseServerSideRedirect)
                 {
+                    HttpContext.Current.Items["IsMedia"] = "true";
                     HttpContext.Current.Server.Transfer(redirect);
                 }
                 else
                 {
+                    Sitecore.Web.WebUtil.AddQueryString(redirect, "IsMedia", "true");
```

Comment:
It seems that this query string is never used.

Reply:
It's alternative flag when Server Redirect is disabled